app/models/curator/filestreams/audio.rb

```diff
@@ -9,5 +9,39 @@ class Filestreams::Audio < Filestreams::FileSet
     has_one_attached :document_access
     has_one_attached :document_master
     has_one_attached :text_plain
+
+    def derivatives_complete?
+      audio_access.attached? && characterization.attached?
+    end
+
+    def derivatives_payload
```

Comment:
Similarly, I wonder if there's a way to DRY this up a bit, where `derivatives_payload` is moved to `Filestreams::FileSet` and takes some args, like `master_attachments` and `required_derivatives`, which could be defined as methods in the individual filestream subclasses.

Or are there too many weird edge cases? Could always override `derivatives_payload` in the subclass for super-edgy cases?

Just a thought.

Also, can we create some specs for this method?